tools/makerootfs.sh

```diff
@@ -42,13 +41,11 @@ do_tar() {
     ARCHARG="--arch ${arch}"
   fi
   if [ -z "$updatetar" ] || [ ! -e "${tarfile}" ]; then
-    # sbom disabled for now; will be re-enabled later
     # shellcheck disable=SC2086
-    linuxkit build --no-sbom --docker ${ARCHARG} --o "${tarfile}" "$ymlfile"
+    linuxkit build --docker ${ARCHARG} --o "${tarfile}" "$ymlfile"
   else
-    # sbom disabled for now; will be re-enabled later
     # shellcheck disable=SC2086
-    linuxkit build --no-sbom --docker ${ARCHARG} --o "${tarfile}.new" --input-tar "${tarfile}"  "$ymlfile"
```

Comment:
@deitch ah! I remember now!!! we enabled it for kernel and rust but LK was not there yet